src/reducers/Entities/NameNoteReducer.js

```diff
@@ -11,13 +11,30 @@ export const NameNoteReducer = (state = initialState(), action) => {
   const { type } = action;
 
   switch (type) {
+    case NAME_NOTE_ACTIONS.UPDATE_DATA: {
+      const { payload } = action;
+      const { data, errors } = payload;
+      const { creatingId } = state;
+
+      const isValid = !(errors?.length > 0);
+
+      const newNameNote = { ...state.creatingById[creatingId], data, isValid };
+      const newById = { ...state.creatingById, [creatingId]: newNameNote };
+
+      return { ...state, creatingById: newById };
+    }
```

Comment:
Think this state is gonna get cleaned up soon, so just went with the flow - I think the `isValid` can probably be at the root of the state